index.js

```diff
@@ -53,7 +53,9 @@ async function run(context, plugins) {
   // Verify config
   await verify(context);
 
+  console.log(options.repositoryUrl)
   options.repositoryUrl = await getGitAuthUrl(context);
+  console.log(options.repositoryUrl)
```

Comment:
forgot `console.log` statements?

Reply:
I added them to debug what's going on on Travis as the build doesn't fail locally.
I'll remove those after.